### DISCUSSION TITLE
oryxBuildBinary? what is this file?

### DISCUSSION THREAD

**Discussion by whtiehack** [community/community#18856] [Codespaces]
![image](https://user-images.githubusercontent.com/12741016/174439173-6ef96c7b-b89c-4afa-b6a0-76aab79f6d53.png)

New codespaces, delete and recreate still there.

![image](https://user-images.githubusercontent.com/12741016/174439480-abd3ac33-0e35-4b44-ac76-384fbbeb85e6.png)

This is my program.. why the codespaces auto build it?

**Comment by Aisuko** [comment]
I have the same question as you because I use the Codespaces too. And I guess it is related to Microsoft Codespaces. As you can see I found the image with the same name with oryxBuildBinary. https://hub.docker.com/_/microsoft-oryx-build

**Reply by whtiehack** [reply]
yes, you are right.

I think space has a default `devcontainer.json` config and it use `https://hub.docker.com/_/microsoft-oryx-build` image as default(just a guess).

such as
https://github.com/microsoft/vscode-dev-containers/tree/v0.236.0/containers/go



`postCreateCommand`
https://github.com/microsoft/vscode-dev-containers/blob/a78a4cb7552a8fe706f3b1e03a40aca2c2b48c7a/containers/go/.devcontainer/devcontainer.json#L38

and uncomment this line, the `oryxBuildBinary` file is gone.
